app/src/features/react-hook-form-wrappers/FormattertTallTextField.tsx

```diff
@@ -0,0 +1,67 @@
+import { TextField, TextFieldProps } from "@navikt/ds-react";
+import { Controller, ControllerProps } from "react-hook-form";
+
+type FormattertTallTextFieldProps = TextFieldProps & {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  control: any;
+  name: string;
+  rules?: ControllerProps["rules"];
+};
+
+/** Et tekstfelt som formaterer innholdet til et tall med tusenvise mellomrom. */
+export const FormattertTallTextField = ({
```

Comment:
![2024-10-11 10 44 41](https://github.com/user-attachments/assets/e912c71d-eae6-46a3-b4f6-480580c85344)

Burde kanskje fått det til å funke med minus også, men det har ikke noe å si for vårt usecase.